src/org/labkey/test/util/FileBrowserHelper.java

```diff
@@ -145,6 +148,41 @@ public void selectFileBrowserItem(@LoggedParam String path)
         }
     }
 
+    public void sortFileBrowserColumn(String columnText, SortDirection sortDirection)
+    {
+        Locator headerLoc = Locator.tagWithClass("div", "fbrowser")
+                .descendant(Locator.tagWithClass("div", "x4-column-header")
+                .withChild(Locator.tagWithClass("div", "x4-column-header-inner")
+                .withChild(Locator.tagWithClass("span", "x4-column-header-text").withText(columnText))));
+
+        WebElement headerElement = headerLoc.findElement(getDriver());
+
+        // often initial state is unsorted, first click sets it to ASC
+        if (!getSortDirection(headerLoc).equals(sortDirection))
+        {
+            doAndWaitForFileListRefresh(()-> headerElement.click());
+        }
+        // if we intended DESC, it may require another click
+        if (!getSortDirection(headerLoc).equals(sortDirection))
+        {
+            doAndWaitForFileListRefresh(()-> headerElement.click());
+        }
+
+        assertThat("Did not get expected sort for column ["+columnText+"]",
+                getSortDirection(headerLoc), is(sortDirection));
+    }
+
+    private SortDirection getSortDirection(Locator headerElementLoc)
+    {
+        String elementClass = headerElementLoc.findElement(getDriver()).getAttribute("class");
+        if (elementClass.contains("column-header-sort-DESC"))
+            return SortDirection.DESC;
+        else if (elementClass.contains("column-header-sort-ASC"))
+            return SortDirection.ASC;
+        else
+            return SortDirection.ASC; // if it doesn't have an explicit sort direction, default behavior is ASC
```

Comment:
Default sort behavior isn't anything in particular. Should return null.
```suggestion
            return null; // No sort defined for column
```